app/src/main/res/values/colors.xml

```diff
@@ -4,4 +4,6 @@
     <color name="colorPrimaryDark">#00574B</color>
     <color name="colorAccent">#D81B60</color>
     <color name="blue">#3A63D3</color>
+    <color name="black">#000000</color>
+    <color name="blackish">#2A2A2A</color>
```

Comment:
Bad naming, if you don't know the color name, then it is better to name it according to its used place, e.g. `main_text_color` or similar.